<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="detectmainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1" columnstretch="1,1">
        <item row="0" column="0">
         <widget class="QGroupBox" name="cameraMainGroupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,2">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
             </property>
             <item>
              <widget class="QTabWidget" name="tabWidget_2">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="tab_4">
                <attribute name="title">
                 <string>Linear Regression Image</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="labelOrigImg">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Table. Linear Regression</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableView" name="tabviewOrig">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12" stretch="5,0">
             <item>
              <widget class="QWidget" name="widgetChart" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButton_5">
                 <property name="text">
                  <string>  Import Image  </string>
                 </property>
                </widget>
               </item>
               <item row="1" column="9">
                <widget class="QPushButton" name="pushButton_8">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="3">
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>  Linear Regression  </string>
                 </property>
                </widget>
               </item>
               <item row="1" column="8">
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="5">
                <widget class="QPushButton" name="pushButton_7">
                 <property name="text">
                  <string>Plot</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="6">
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="7">
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Detect</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="10">
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="cameraMainlVLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
             </property>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,2">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QTabWidget" name="tabWidget">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="tabPosition">
                  <enum>QTabWidget::TabPosition::North</enum>
                 </property>
                 <property name="tabShape">
                  <enum>QTabWidget::TabShape::Rounded</enum>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="tab_3">
                  <attribute name="title">
                   <string>Detection Image</string>
                  </attribute>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="labelRecgImg">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Shadow::Raised</enum>
                   </property>
                   <property name="text">
                    <string>Progress:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="progressBar">
                   <property name="value">
                    <number>100</number>
                   </property>
                   <property name="textVisible">
                    <bool>false</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="invertedAppearance">
                    <bool>false</bool>
                   </property>
                   <property name="textDirection">
                    <enum>QProgressBar::Direction::BottomToTop</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Shadow::Raised</enum>
                   </property>
                   <property name="text">
                    <string>Period:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="lcdNumber">
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::Panel</enum>
                   </property>
                   <property name="intValue" stdset="0">
                    <number>377</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Table. Detection</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableView" name="tabviewRecg">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
